course/migrations/0047_delete_duplicate_enrollments.py

```diff
@@ -10,18 +10,19 @@ def delete_duplicates(apps, schema_editor):
     unique_fields = ['course_instance', 'user_profile']
 
     duplicates = (
-        Enrollment.objects.values(*unique_fields)
-        .order_by()
-        .annotate(min_id=Min('id'), count_id=Count('id'))
-        .filter(count_id__gt=1)
-        )
+        Enrollment.objects
+            .values(*unique_fields)
+            .order_by()
+            .annotate(min_id=Min('id'), count_id=Count('id'))
+            .filter(count_id__gt=1)
+    )
```

Comment:
What do you personally think about this douple deindentation? This could be worse than the other option too... I'm personally really not sure anymore :disappointed: 

A good counter example:

```python
bar = (
    foo
        .something()
        .another()
)
```

Reply:
Your counter example looks a bit strange and there I would probably use just single indentation. I'm not sure about the double indentation, I think it looks good in that case (probably only because there is Enrollment.objects and not just Enrollment on the first line). We could go with the option 3.